### ISSUE TITLE
Feat: Import Swagger Endpoints

### ISSUE THREAD

**Issue by walterstypula** [frigus02/RESTer#536]
I really love this tool and have been using it for a long time, maybe 4 or 5 years now. I much prefer it over postman. However, there's only one reason I have postman installed: to import a swagger file. Only to then export it to import it into RESTer. I wish we could import swagger file directly into RESTer.

Thanks,

**Comment by frigus02** [comment]
Yeah, that would be a great feature.

I'm not sure I can get to this anytime soon. But I'd definitely review a PR, if you or someone else would like to work on this. I believe to support data import, a file with a "parse" function, similar to https://github.com/frigus02/RESTer/blob/master/src/background/exportImport/providers/postman.js should mostly be enough.

**Comment by walterstypula** [comment]
Yeah, I figured it could be a good item added to the backlog. I'll try to look into it, I'm not the best JS developer.
